bin/init

```diff
@@ -62,4 +61,4 @@ setup_database() {
 
 setup_database
 bundle exec rake db:migrate
-bundle exec "rails s -b 0.0.0.0 -p $VELUM_PORT --pid /tmp/puma.pid Puma"
```

Comment:
This is the removal of the previous command that we were using, we can keep the discussion on the previous `0.0.0.0` bind thread :)